playbook-website/app/javascript/components/App.tsx

```diff
@@ -1,17 +1,29 @@
 import React, { useState, useEffect } from "react"
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom"
-import MainSidebar from "./MainSidebar"
-import { Layout, Background, Icon, Title, Body, Button, Flex, FlexItem } from "playbook-ui"
-import HeaderMobile from "../images/pb_generic_header.jpg"
-import HeaderDesktop from "../images/pb_generic_header_desktop.jpg"
-import Footer from "./Footer"
-import DarkModeToggle from "./DarkModeToggle"
+import { Outlet } from "react-router-dom"
+import ReactSidebar from "./ReactSidebar"
```

Comment:
React seems to be unimportant here. Can we drop/change that? I'd be more curious to define this as the site navigation or something more descriptive.

Reply:
Sure. I can change that.